viewing_party/party.py

```diff
@@ -1,23 +1,117 @@
+import operator
+
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie = {}
+
+    if not title or not genre or not rating:
+        return None
+    
+    movie['title'], movie['genre'], movie['rating'] = title, genre, rating
+    
+    return movie
+
+def add_to_watched(user_data, movie):
+    user_data['watched'].append(movie)
+    
+    return user_data

```

Comment:
👍  While this works, what if the `movie` being passed in isn't valid (missing keys, wrong data type, etc.). A helper function to sanitize the inputs would be helpful here!